files/zh-cn/web/css/css_grid_layout/grid_layout_and_progressive_enhancement/index.md

```diff
@@ -9,46 +9,21 @@ slug: Web/CSS/CSS_grid_layout/Grid_layout_and_progressive_enhancement
 
 ## 浏览器支持
 
-除了在 Internet Explorer 中，CSS Grid Layout 在 Safari，Chrome，Opera，Firefox 和 Edge 中都没有前缀。对这些指南中详述的所有属性和值的支持可跨浏览器进行互操作。这意味着如果您在 Firefox 中编写一些网格布局代码，它应该在 Chrome 中以相同的方式工作。这不再是实验规范，您可以安全地在生产环境中使用它。.
-
-## IE 和 Edge
-
-应当谨记 CSS 网格布局在 IE 上实现的最早版本是 IE10。这个早期规范并没有包含最新规范的所有属性和可用值。即使是相同的属性和相同的值，在 IE10 中的实现和现行的规范也有很大差异。早期规范也是 Edge15 之前实现的网格布局的版本。
-
-IE/Edge(≤15)版本的规范带有`-ms`前缀，IE/Edge(≤15) 上实现的属性如下：
-
-- {{cssxref("grid-template-columns")}} 变成 `-ms-grid-columns`
-- {{cssxref("grid-template-rows")}} 变成 `-ms-grid-rows`
-- {{cssxref("grid-row-start")}} 变成 `-ms-grid-row`
-- {{cssxref("grid-column-start")}} 变成 `-ms-grid-column`
-- {{cssxref("align-self")}} 变成 `-ms-grid-row-align`
-- {{cssxref("justify-self")}} 变成 `-ms-grid-column-align`
-
-IE 版本另有一些在新版规范中非必须的属性`-ms-grid-column-span`和`-ms-grid-row-span`。这个版本不包含自动定位能力和网格模板区域。从 IE10 到 Edge15 的版本，使用`-ms`属性可以实现一些简单的网格布局。由于这些属性包含厂商前缀，它们在任何支持最新规范使用无前缀属性的浏览器上将无法生效。
-
-### Autoprefixer 对网格布局的支持
-
-流行的*[Autoprefixer](https://github.com/postcss/autoprefixer)*更新之后已经能够支持`-ms-`网格布局版本。网格前缀默认是禁用的，你可以通过`grid: true`选项启用它。
-
-```
-autoprefixer({ grid: 'autoplace' })
-```
-
-Grid 网格前缀被默认禁用是因为存在一些属性不能添加前缀。
+CSS Grid Layout 在 Safari，Chrome，Opera，Firefox 和 Edge 中都没有前缀。对这些指南中详述的所有属性和值的支持可跨浏览器进行互操作。这意味着如果您在 Firefox 中编写一些网格布局代码，它应该在 Chrome 中以相同的方式工作。这不再是实验规范，您可以安全地在生产环境中使用它。
 
 ## 我的布局使用 CSS 网格布局安全吗？
 
-无论使用任何前端技术，是否使用 CSS 网格布局都取决于网站访问者通常使用的浏览器。如果他们更多地使用最新版本地 Firefox，Chrome，Opera 和 Safari，那么在这些浏览器更新之后开始使用 CSS 网格就很有意义了。但是如果你的站点服务于一个更旧浏览器绑定的超市，那目前就没有必要。然而，我的建议是不要根据过去浏览器中新规范的推出方式来进行假设。CSS 网格布局是不同的，这取决它已经处于开发阶段的时间，和不同浏览器厂商为了确保所发布的内容对所有人都能以相同方式工作所做的工作。
+是的。无论使用任何前端技术，是否使用 CSS 网格布局都取决于网站访问者通常使用的浏览器。
 
-## Starting to use Grid in production
+## 开始在生产中使用网格布局
 
-It is worth noting that you do not have to use grid in an _all or nothing_ way. You could start by simply enhancing elements in your design with grid, that could otherwise display using an older method. Overwriting of legacy methods with grid layout works surprisingly well, due to the way grid interacts with these other methods.
+值得注意的是，您不必以全有或全无的方式使用网格。首先使用网格增强设计中的元素，否则这些元素可能会以旧的方式显示。由于网格与这些其他方法交互的方式，使用网格布局覆盖遗留方法的效果出奇地好。
 
-### Floats
+### 浮动
 
-We have typically used [floats](/zh-CN/docs/Learn/CSS/CSS_layout/Floats) to create multiple column layouts. If you have floated an item, which is also a grid item in a supporting browser, the float will no longer apply to the item. The fact is that _a grid item takes precedence._ In the example below, I have a simple media object. In a non-supporting browser, I use {{cssxref("float")}}, however I have also defined the container as a grid container, in order to use the alignment properties that are implemented in CSS Grids.
+[Floats](/zh-CN/docs/Learn/CSS/CSS_layout/Floats) 曾经用于创建多列布局。如果您在旧代码库使用了浮动布局，则不会发生冲突。网格项目会忽略浮动属性；事实上，网格项目优先级更高。在下面的示例中，我们有一个简单的 media 对象。如果 float 没有从旧版 CSS 中删除，且容器是网格容器，也不会有问题。我们可以使用 CSS 网格中实现的对齐属性。
```

Comment:
Note here, we do not use [plural forms](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%A4%8D%E6%95%B0%E5%BD%A2%E5%BC%8F).

```suggestion
[浮动](/zh-CN/docs/Learn/CSS/CSS_layout/Floats) 曾经用于创建多列布局。如果您在旧代码库使用了浮动布局，则不会发生冲突。网格项目会忽略浮动属性；事实上，网格项目优先级更高。在下面的示例中，我们有一个简单的 media 对象。如果 float 没有从旧版 CSS 中删除，且容器是网格容器，也不会有问题。我们可以使用 CSS 网格中实现的对齐属性。
```